src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -74,6 +74,7 @@
     <PackageReference Include="KubeClient" Version="2.1.5" />
     <PackageReference Include="KubeClient.Extensions.KubeConfig" Version="2.1.5" />
     <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.1.1" />
+    <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.3.1" />
```

Comment:
For now, let's target `1.3.2-rc` since that contains all the new operations we recently added. Before we actually merge this, we should update it again to `1.3.2` (the final released version).